apps/ergw_core/src/gtp_context_reg.erl

```diff
@@ -117,13 +112,18 @@ handle_cast({link, Pid}, State) ->
 handle_cast(_Msg, State) ->
     {noreply, State}.
 
-handle_info({'EXIT', Pid, _Reason}, State0) ->
-    Objs = ets:lookup(?SERVER, Pid),
-    [global_del_key(Key, {Handler, Pid}) || {_, {Handler, Key}} <- Objs],
-    ets_delete_objects(?SERVER, Objs ++ mk_reg_pids(Objs)),
+handle_info({'EXIT', Pid, _Reason}, State) ->
+    Notify = maps:get(Pid, State, []),
+    proc_lib:spawn(
```

Comment:
I still think proc_lib is overkill in this case, but it's not a big deal.
The fun can be finished with any value, there's no need to return an ok since nobody gets it.